source/Grid/defaultCellRangeRenderer.js

```diff
@@ -40,7 +97,8 @@ export default function defaultCellRangeRenderer ({
         isScrolling,
         key,
         rowIndex,
-        style
+        style,
+        visibility: findOverlap(gridDimensions, style)
```

Comment:
> This is probably the single area where perf matters

Perf is an extremely part of this lib, and I dont want to mess that up!

I think there are two questions here:

1. What does visible mean?
Does it mean the item is fully displayed? Partly displayed? In the use case where a dev may want to render just the "cheaper" parts of the cell, they need to know how much of the cell will be visible, as they will render things differently if 0.03 is visible vs if 0.98 is visible.

2. When is visibility important?
The purpose of visibility is to display a cheaper version of the cell _when it is visibile_. Arguably, this also means _when the cell is at rest_, as if it were scrolling the dev would use `isScrolling` and render an even cheaper version of the cell! Hence, the simplest solution might be not to test or return visibility while a cell is being scrolled:
`visibility: isScrolling ? {x: null, y: null} : findOverlap(gridDimensions, style)`